doctest/extensions/mpi_sub_comm.h

```diff
@@ -1,4 +1,5 @@
-#pragma once
+#ifndef DOCTEST_SUB_COMM_H
+#define DOCTEST_SUB_COMM_H
```

Comment:
Generally I prefer pragma once, but I dont use it for the same reason.

```suggestion
#ifndef DOCTEST_MPI_SUB_COMM_H
#define DOCTEST_MPI_SUB_COMM_H
```